ipad/Database/Storage.swift

```diff
@@ -35,7 +35,7 @@ class Storage {
         let shifts = Storage.shared.getSyncableItems()
         var invalidShifts: [ShiftModel] = []
         for shift in shifts {
-            if shift.shitEndComments.count > 300 || shift.shitStartComments.count > 300 {
+            if shift.shiftEndComments.count > 300 || shift.shiftStartComments.count > 300 {
```

Comment:
One of the many examples of renaming for better readability and less confusion

Reply:
🤦‍♂️